processors/arxiv/docling_extractor.py

```
@@ -619,10 +627,20 @@
         metadata_path = output_subdir / f"{arxiv_id.replace('/', '_')}_meta.json"
         metadata_path.write_text(json.dumps(metadata, indent=2), encoding='utf-8')
         
+        # Parse sections from markdown for structure (simple parsing)
+        sections = []
+        for line in markdown.split('\n'):
+            if line.startswith('#'):
+                level = len(line) - len(line.lstrip('#'))
+                title = line.lstrip('#').strip()
+                if title:
+                    sections.append({'level': level, 'title': title})
+        
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)